src/config/dynamic-theme-fixes.config

```diff
@@ -3274,6 +3274,18 @@ CSS
 
 ================================
 
+feedly.com
+
+CSS
+.entry.u0:hover {
+    background-color: rgb(35, 35, 35) !important;
+}
+.entry--selected.u0, .entry--selected.u0:hover, button.secondary:hover {
+    background-color: rgb(50, 50, 50) !important;
```

Comment:
Please change it to `background-color: ${#cdcdcd} !important;`

Reply:
OK! I used ccc instead of cdcdcd to make it a tiny bit brighter; cdcdcd was a too dark compared with the normal background (unselected/unhovered)